test/number.js

```diff
@@ -209,15 +209,15 @@ describe('Parser → Number : Loose', () => {
     },
     {
       test: '5+5',
-      expected: { value: '5', unit: '', length: 3 }
+      expected: { value: '+5', unit: '', length: 2 }
```

Comment:
@shellscape break, `5+5` should be as `5 + 5`

Reply:
@evilebottnawi thanks for taking a look. can you elaborate? I'm not sure what you mean there.

Reply:
@shellscape It is difficult to say for sure, since we have rather complicated logic around `+`, `-` and 
 `/`. I will test this branch tomorrow and leave feedback here :+1:

Reply:
Thank you!